core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -763,21 +760,19 @@ public synchronized void startInternal() throws IOException {
         )
         .collect(Collectors.toSet());
     mRaftGroup = RaftGroup.valueOf(RAFT_GROUP_ID, peers);
-    initServer();
-    super.registerMetrics();
     LOG.info("Starting Raft journal system. Cluster addresses: {}. Local address: {}",
         mClusterAddresses, mLocalAddress);
-    long startTime = System.currentTimeMillis();
     try {
+      initServer();
+      long startTime = System.currentTimeMillis();
       mServer.start();
+      LOG.info("Started Raft Journal System in {}ms", System.currentTimeMillis() - startTime);
     } catch (IOException e) {
```

Comment:
seems like Ratis document doesn't say much about what the IOexception is about so it's hard to say it's retryable or not...

Reply:
I deemed it retryable because it seems feasible to retry creating a Ratis server. We could label it unretryable to be safe. What do you think?